### ISSUE TITLE
Deleting sessions sometimes not recognized

### ISSUE THREAD

**Issue by irvnriir** [tim-we/tabs-aside#87]
Deleting Sessions at bookmarks or sidebar, sometimes are not recognized (so after you 'hide' the tabs, they go into not existing Session and disappear) .

**Comment by irvnriir** [comment]
It seems that this is actually about doing so in bookmarks, and related to https://github.com/tim-we/tabs-aside/issues/86 .

**Comment by tim-we** [comment]
Yes it seems so. However, trying to set tabs aside into a session that does not exist anymore should throw an error and not remove that tab. I'll look into that.
